roles/openshift_health_checker/openshift_checks/logging/logging_index_time.py

```diff
@@ -0,0 +1,168 @@
+"""
+Check for ensuring logs from pods can be queried in a reasonable amount of time.
+"""
+
+import json
+import time
+
+from uuid import uuid4
+
+from openshift_checks import get_var, OpenShiftCheckException
+from openshift_checks.logging.logging import LoggingCheck
+
+
+ES_CMD_TIMEOUT_SECONDS = 30
+
+
+class LoggingIndexTime(LoggingCheck):
+    """Check that pod logs are aggregated and indexed in ElasticSearch within a reasonable amount of time."""
+    name = "logging_index_time"
+    tags = ["health", "logging"]
+
+    logging_namespace = "logging"
+
+    def run(self, tmp, task_vars):
+        """Add log entry by making unique request to Kibana. Check for unique entry in the ElasticSearch pod logs."""
+        try:
+            log_index_timeout = int(
+                get_var(task_vars, "openshift_check_logging_index_timeout_seconds", default=ES_CMD_TIMEOUT_SECONDS)
+            )
+        except ValueError:
+            return {
+                "failed": True,
+                "msg": ('Invalid value provided in your Ansible inventory file for '
+                        '"openshift_check_logging_index_timeout_seconds". Value must '
+                        'be an integer representing an amount in seconds.'),
+            }
```

Comment:
I think that it might be worth the price of the added annoyance by having the check fail. That way, a user is at least aware of why a value supplied for `openshift_check_logging_index_timeout_seconds` was invalid